srcpkgs/go/template

```diff
@@ -10,7 +10,7 @@ maintainer="Michael Aldridge <maldridge@VoidLinux.eu>"
 homepage="http://golang.org/"
 license="3-clause-BSD"
```

Comment:
BSD-3-Clause

Reply:
Dam that was quick XD

Fixed it thanks :3